src/main/java/dinkplugin/message/DiscordMessageHandler.java

```diff
@@ -167,6 +170,7 @@ private static NotificationBody<?> injectContent(@NotNull NotificationBody<?> bo
 
         Author author = Author.builder()
             .name(body.getPlayerName())
+            .url(body.getPlayerUrl())
```

Comment:
Would become: `playerLookupService.playerUrl(body.getPlayerName())`

Reply:
I didn't directly see the reason why `injectContent` was `private static`? I removed it to access the `config` inside `DiscordMessageHandler`. 

Just a headsup incase I missed something why it should be static.

Reply:
The `static` was more stylistic / for ease of testing at a later date.

If removing `static`, `footerText`/`footerIcon` can also be accessed directly from config (rather than being passed as args)... but I'm okay with refactoring this in a later PR